sgl-kernel/tests/test_sampling.py

```diff
@@ -15,7 +15,7 @@ def test_top_k_top_p_joint_sampling_from_probs(batch_size, vocab_size, p):
     elif p == 0.5:
         k = int(vocab_size * 0.1)
     else:
-        raise ValueError("p not recognized")
+        raise ValueError("pp not recognized")
```

Comment:
What is the purpose of changing this?

Reply:
NVM this pr is just to poke the CI to see if the latest sgl-kernel building works on CI and it does. There seems to be some issues on my side.

Reply:
I see. Thanks!